job/replay_manager.go

```diff
@@ -30,6 +30,8 @@ var (
 	TimestampLogFormat = "2006-01-02T15:04:05+00:00"
 	// schedulerBatchSize number of run instances to be checked per request
 	schedulerBatchSize = 100
+	//replayListWindow window interval to fetch recent replays
+	replayListWindow = -30 * 24 * time.Hour
```

Comment:
let's make it 3 months at least as we won't have many replays anyway.

Reply:
updated